test/plot_fromtree.py

```diff
@@ -113,55 +112,44 @@ def select_all_but_one(cutstring):
-    h_base[sample_name+list_histos[16]] = ROOT.TH1F(sample_name+list_histos[16], "B-tag of 4th jet", 13, 0.89, 1.)
-    h_base[sample_name+list_histos[17]] = ROOT.TH1F(sample_name+list_histos[17], "m_{12} invariant mass", 15, 0., 600.)
-    h_base[sample_name+list_histos[18]] = ROOT.TH1F(sample_name+list_histos[18], "m_{34} invariant mass", 15, 0., 600.)
+    h_base[sample_name+list_histos[13]] = ROOT.TH1F(sample_name+list_histos[13], "B-tag of 1st jet", 15, 0.89, 1.)
+    h_base[sample_name+list_histos[14]] = ROOT.TH1F(sample_name+list_histos[14], "B-tag of 2nd jet", 15, 0.89, 1.)
+    h_base[sample_name+list_histos[15]] = ROOT.TH1F(sample_name+list_histos[15], "B-tag of 3rd jet", 15, 0.89, 1.)
+    h_base[sample_name+list_histos[16]] = ROOT.TH1F(sample_name+list_histos[16], "B-tag of 4th jet", 15, 0.89, 1.)
+    h_base[sample_name+list_histos[17]] = ROOT.TH1F(sample_name+list_histos[17], "No. of primary verticies", 30, 0., 30.)
 
 h_QCD[list_histos[0]]  = ROOT.TH1F(list_histos[0], "p_{t} of 1st jet", 25, 50., 500.)
 h_QCD[list_histos[1]]  = ROOT.TH1F(list_histos[1], "p_{t} of 2nd jet", 25, 50., 500.)
 h_QCD[list_histos[2]]  = ROOT.TH1F(list_histos[2], "p_{t} of 3rd jet", 25, 50., 500.)
 h_QCD[list_histos[3]]  = ROOT.TH1F(list_histos[3], "p_{t} of 4th jet", 25, 50., 500.)
 h_QCD[list_histos[4]]  = ROOT.TH1F(list_histos[4], "#Delta_{#phi} of the two jet pairs", 15, 0., 3.14)
-h_QCD[list_histos[5]]  = ROOT.TH1F(list_histos[5], "#Delta_{#eta} of the two jet pairs", 15, -5., 5.)
+h_QCD[list_histos[5]]  = ROOT.TH1F(list_histos[5], "#Delta_{#eta} of the two jet pairs", 20, -5., 5.)
 h_QCD[list_histos[6]]  = ROOT.TH1F(list_histos[6], "p_T of the first jet pair", 50, 0., 500.)
 h_QCD[list_histos[7]]  = ROOT.TH1F(list_histos[7], "p_T of the second jet pair", 50, 0., 500.)
-h_QCD[list_histos[8]]  = ROOT.TH1F(list_histos[8], "4jets invariant mass", 13, 250., 1000.)
+h_QCD[list_histos[8]]  = ROOT.TH1F(list_histos[8], "4jets invariant mass", 40, 0., 1000.)
 h_QCD[list_histos[9]]  = ROOT.TH1F(list_histos[9], "#eta of 1st jet", 20, -7., 7.)
 h_QCD[list_histos[10]] = ROOT.TH1F(list_histos[10], "#eta of 2nd jet", 20, -7., 7.)
 h_QCD[list_histos[11]] = ROOT.TH1F(list_histos[11], "#eta of 3rd jet", 20, -7., 7.)
 h_QCD[list_histos[12]] = ROOT.TH1F(list_histos[12], "#eta of 4th jet", 20, -7., 7.)
-h_QCD[list_histos[13]] = ROOT.TH1F(list_histos[13], "B-tag of 1st jet", 13, 0.89, 1.)
-h_QCD[list_histos[14]] = ROOT.TH1F(list_histos[14], "B-tag of 2nd jet", 13, 0.89, 1.)
-h_QCD[list_histos[15]] = ROOT.TH1F(list_histos[15], "B-tag of 3rd jet", 13, 0.89, 1.)
-h_QCD[list_histos[16]] = ROOT.TH1F(list_histos[16], "B-tag of 4th jet", 13, 0.89, 1.)
-h_QCD[list_histos[17]] = ROOT.TH1F(list_histos[17], "m_{12} invariant mass", 15, 0., 600.)
-h_QCD[list_histos[18]] = ROOT.TH1F(list_histos[18], "m_{34} invariant mass", 15, 0., 600.)
-
-h_ma1_ma2 = ROOT.TH2F("h_ma1_ma2", "m_{12} vs m_{34}", 15, 0., 600., 15, 0., 600.)
```

Comment:
I have rechecked the code, it seems that you have updated the code before it was given to me or there might be some other reason. Every time I see this code, some specific information appears to be missing (e.g particular histograms related etc.) which should not be the case of mistake.